internal/pkg/mojang/mojang.go

```diff
@@ -0,0 +1,226 @@
+package mojang
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+)
+
+const (
+	userAgentFormat = "mworzala/mc/%s"
```

Comment:
if reused, this should be moved somewhere common so its not redefined here and modrinth (probably like `util.MakeUserAgent(idVersion)`